src/SvgIcon/SvgIcon.js

```diff
@@ -39,6 +39,7 @@ export default function SvgIcon(props, context) {
     <svg
       className={className}
       viewBox={viewBox}
+      aria-hidden="true"
```

Comment:
That's 20% of the use case following my previous 80%. I agree we miss a title property!